Kernel/VirtIO/VirtIOConsole.h

```diff
@@ -1,51 +1,77 @@
 /*
- * Copyright (c) 2021, the SerenityOS developers.
+ * Copyright (c) 2021, Kyle Pereira <hey@xylepereira.me>
```

Comment:
Just so that you don't think I'm egotistical enough to go around removing other people's copyright headers on purpose :P — I should clarify that the original `VirtIOConsole.h` was moved to `VirtIOConsolePort.h` and I did rewrite this file from scratch. Of course it's pretty derivative so I will definitely add the previous copyright header :^)